tests/transformers/test_transformers_model_export.py

```diff
@@ -2697,6 +2689,7 @@ def test_instructional_pipeline_with_prompt_in_output(model_path):
                 "inputs": '[{"type": "string", "name": "question"}, {"type": "string", '
                 '"name": "context"}]',
                 "outputs": '[{"type": "string"}]',
+                "params": None,
```

Comment:
This is a local-only test. Keeping this up to date so that all local tests pass when modifying this suite.